locales/en/translation.json

```diff
@@ -780,7 +780,7 @@
         "formErrorMessage": {
           "errorSummaryTitleText": "There is a problem",
           "errorSummaryDescriptionText": "Select yes if you want to delete your details and prove your identity again",
-          "errorRadioMessage": "details and prove your identity again details and prove your identity again"
+          "errorRadioMessage": "Select yes if you want to delete your details and prove your identity again"
```

Comment:
do we know if the welsh translation is correct or needs re-requesting?

Reply:
The welsh translations looked correct, I did check. Was odd that only the english was out.